content/manuals/build/bake/remote-definition.md

```diff
@@ -6,15 +6,33 @@ keywords: build, buildx, bake, file, remote, git, http
 
 You can build Bake files directly from a remote Git repository or HTTPS URL:
```

Comment:
I think the only change we need is to link to Build context documentation to have more detail if this is what is confusing: https://docs.docker.com/build/concepts/context/#git-repositories

```suggestion
You can build Bake files directly from a remote [Git repository](../concepts/context.md#git-repositories):
```